src/Pages/ChannelRequests.razor

```diff
@@ -9,11 +9,18 @@
 
 <PageTitle>Channel Operation Requests</PageTitle>
 
-@if (!_hidePendingRequests)
```

Comment:
What would be render if there are no requests, an empty datagrid?

Reply:
For finance managers will render the requests waiting for signature.
For node managers will render the requests the node manager created and are pending signature from the finance manager.
For admins it won't render at all. All requests will show in the other datagrid

The thing is that i needed the "new" button for node managers, but the button needs the datagrid to exist, so i had to separate them into pending and approved for node managers too